proto/config/patch_envelope.proto

```diff
@@ -0,0 +1,75 @@
+// Copyright(c) 2023 Zededa, Inc.
+// All rights reserved
+
+syntax = "proto3";
+
+package org.lfedge.eve.config;
+option go_package = "github.com/lf-edge/eve/api/go/config";
```

Comment:
This update broke eve-api, as it references `lf-edge/eve/api` and not `lf-edge/eve-api`. It does it in one other place as well.

I will put in place a PR to fix it.

Reply:
Fixed in #7